static/app/components/footer.tsx

```diff
@@ -68,7 +68,7 @@ const FooterLink = styled(ExternalLink)`
 `;
 
 const LogoLink = styled(props => (
-  <a href="/" tabIndex={-1} {...props}>
+  <a href="https://sentry.io/welcome/" target="_blank" tabIndex={-1} {...props}>
```

Comment:
Let's use `<ExternalLink>` component here

Reply:
Done in 7492de0bd8b0728e3e7d0f5196b9ad01e3873dc4.